exercises/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,7 @@
 // Start by creating a variable `message`
 
-console.log(message);
+var name = "Aisha"; 
+var nameLength = name.length;
+var message = "My name is " + name + " and the length of name is " +nameLength +" characters long."
```

Comment:
Maybe we could do with a pronoun just before name on " and the length of name is " and a space before nameLength.